com.unity.uiwidgets/Runtime/engine/UIWidgetsMessageManager.cs

```diff
@@ -74,18 +74,28 @@ public class UIWidgetsMessageManager: MonoBehaviour {
             }  
         }
 
+        Queue<string> messages = new Queue<string>();
+
         void OnUIWidgetsMethodMessage(string message) {
```

Comment:
this method should only work on Android ? if so we should use #define to wrap it in.